pages/about/translating.md

```diff
@@ -25,11 +25,13 @@ footer: |
 {% include box.html type="start" title="Summary" class="" %}
 {:/}
 
-This page provides general instructions on translating WAI resources.
+Help make Web Accessibility Initiative (WAI) resources available in your language.
 
-For a list of existing translations, see {% include link.html to="/translations/" text="All WAI Translations" %}.
+Join the volunteer translation community to raise awareness of digital accessibility among a global audience. This page provides general instructions on translating WAI standards and resources.
```

Comment:
I'm thinking that 'This page provides general instructions on translating WAI standards and resources.' is the main summary of this page. Perhaps it should be the first sentence? Or, at least in a paragraph of it's own here so it stands out.